packages/waku/src/lib/utils/stream.ts

```diff
@@ -31,6 +56,13 @@ export const streamToString = async (
   return outs.join('');
 };
 
+export function arrayBufferToString(buffer: ArrayBuffer): string {
+  const uint8Array = new Uint8Array(buffer);
+  return Array.from(uint8Array)
+    .map((byte) => String.fromCharCode(byte))
```

Comment:
This doesn't seem to work for non-ASCII chars.

Do we have the same issue in `parseFormData`?

Reply:
yep